Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04 as builder
+ENV DEBIAN_FRONTEND noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN true
+RUN echo 'tzdata tzdata/Areas select Etc' | debconf-set-selections; \
+    echo 'tzdata tzdata/Zones/Etc select UTC' | debconf-set-selections; \
+    apt-get update && apt-get upgrade -y && apt-get install --no-install-recommends -y \
+    cmake \
+    ninja-build \
+    gcc \
+    g++ \
+    maven \
+    flex \
+    bison \
+    libxml2-utils \
+    dpkg-dev
+COPY . /tmp/cbmc
+WORKDIR /tmp/cbmc
+RUN cmake -S . -Bbuild -G Ninja -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER=/usr/bin/gcc -DCMAKE_CXX_COMPILER=/usr/bin/g++ && cd build; ninja -j2
+
+FROM ubuntu:20.04 as runner
+COPY --from=builder /tmp/cbmc/build/bin/* /usr/local/bin/
+WORKDIR /tmp/cbmc/
```

Comment:
It is definitely needed it seems - without it we get a CMake errors that the current directory doesn't contain a `CMakeLists.txt`.

It is also recommended by the [docker documentation to always use `WORKDIR` instead of `cd` or something else](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#workdir)

Reply:
@NlightNFotis I don't understand how that could be the case given this command is on the container that doesn't even have cmake installed.

Reply:
@hannes-steffenhagen-diffblue You are right. Apologies for that, I got confused with the one in line 17. This definitely seems extraneous.